### ISSUE TITLE
Allow additional shortcuts for submitting gum write

### ISSUE THREAD

**Issue by annervisser** [charmbracelet/gum#423]
**Is your feature request related to a problem? Please describe.**
While <kbd>Ctrl</kbd>+<kbd>d</kbd> is a unixy way of exiting a program, it doesn't feel logical to me as a "submit".

**Describe the solution you'd like**
I'd love some additional shortcuts for saving my text, <kbd>Ctrl</kbd>+<kbd>s</kbd> and <kbd>Ctrl</kbd>+<kbd>x</kbd> (Nano) come to mind.

It would also be helpful if the shortcut(s) were listed in the description of `gum write --help`

**Describe alternatives you've considered**
n/a

**Additional context**
Current documentation: 
> Note: CTRL+D is used to complete text entry. CTRL+C and esc will cancel.

Current output of `gum write --help`:
```
Usage: gum write

Prompt for long-form text

Flags:
 [...]

Style Flags
 [...]
```

**Comment by octoshrimpy** [comment]
second this. shift-enter is another one I recommend. relevant to #286 and https://github.com/charmbracelet/gum/pull/433

**Comment by caarlos0** [comment]
I believe shift+enter requires kitty keyboard, which is coming on v2 (cc/ @aymanbagabas)

that said, I also don't like ctrl+d being used for this, but I agree ctrl+s and ctrl+x are better options

**Comment by meowgorithm** [comment]
yep <kbd>shift+enter</kbd> will be possible in with Bubble Tea v2, however also keep in mind that in many terminals and multiplexers the feature simply won't be available (at least not by default). We can offer that when it _is_ available.

Anyway, it will probably be helpful to show the options on the onscreen help (`--show-help`).

One question: isn't "submit" currently just <kbd>enter</kbd>?

**Comment by caarlos0** [comment]
> One question: isn't "submit" currently just enter?

you're right!

but there's another 'weirdness': ctrl+j is used to enter a new line, which I also don't like (and conflicts with my tmux mappings)... would probably bind that to shift+enter as well on v2, which seems to be more easy to figure out
